db/warehouse/migrate/20250226145730_create_idx_on_youth_eds.rb

```diff
@@ -0,0 +1,10 @@
+class CreateIdxOnYouthEds < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+  def change
+    add_index 'YouthEducationStatus', ['EnrollmentID', 'data_source_id', 'PersonalID'], name: 'idx_youth_eds_hud_keys', algorithm: :concurrently
```

Comment:
`algorithm: :concurrently` makes strong migration happy, but often causes issues on deployment.  We should keep an eye on it.

Reply:
Yeah, maybe we just want to lock YouthEducationStatus instead. I guess it does seem to build pretty quickly even for a few million rows.